dokka-subprojects/plugin-base/src/test/kotlin/signatures/SignatureTest.kt

```diff
@@ -1365,6 +1366,59 @@ class SignatureTest : BaseAbstractTest() {
         }
     }
 
+    @Test
+    @OnlySymbols("context parameters")
+    fun `fun with context parameters`() = withContextParametersEnabled {
+        val source = source("""
+            context(s: String, _:Int)
+            fun Int.simpleFun(a: Int): String = \"\""
+        """.trimIndent())
+        val writerPlugin = TestOutputWriterPlugin()
+
+        testInline(
+            source,
+            configuration,
+            pluginOverrides = listOf(writerPlugin)
+        ) {
+            renderingStage = { _, _ ->
+                writerPlugin.writer.renderedContent("root/example/simple-fun.html").firstSignature().match(
+                    "context(", @OptIn(ExperimentalDokkaApi::class) ContextParameters(
```

Comment:
minor: I think it will be enough to just guard the function with OptIn, so that this part of the code will be less cluttered (same in other similar places)